gunrock/graphio/market.cuh

```diff
@@ -327,7 +327,8 @@ int ReadMarketStream(
 
 		if (ll_row <  1 || ll_col <  1)
 		{
-		    fprintf(stderr, "\nError Invalid graph: Bad MTX format, indices cannot start with 0.\n");
+		    fprintf(stderr, "\nError Invalid graph: Bad MTX format, indices
```

Comment:
Pretty sure you don't want a newline in the middle of a string. I think what you want is

```
fprintf(stderr, "start of text "
        "next line of text");
```